Engine_Revit_UI/Convert/Common/ToBHoM/Material.cs

```diff
@@ -98,5 +109,44 @@ internal static oM.Common.Materials.Material ToBHoMMaterial(this Material materi
         }
 
         /***************************************************/
+        /****             Private methods               ****/
+        /***************************************************/
+
+        private static void Update(this oM.Common.Materials.Material material_Destination, Material material_Source)
+        {
+            if (material_Source == null)
+                return;
+
+            ElementId aElementId = material_Source.StructuralAssetId;
+            if (aElementId == null || aElementId == ElementId.InvalidElementId)
+                return;
+
+            Document aDocument = material_Source.Document;
+
+            PropertySetElement aPropertySetElement = aDocument.GetElement(aElementId) as PropertySetElement;
+
+            StructuralAsset aStructuralAsset = aPropertySetElement.GetStructuralAsset();
+
+            if (aStructuralAsset.Behavior != StructuralBehavior.Isotropic)
+                return;
+
+            material_Destination.YoungsModulus = UnitUtils.ConvertFromInternalUnits(aStructuralAsset.YoungModulus.X, DisplayUnitType.DUT_PASCALS);
+            material_Destination.Density = UnitUtils.ConvertFromInternalUnits(aStructuralAsset.Density, DisplayUnitType.DUT_KILOGRAMS_PER_CUBIC_METER);
+            material_Destination.CoeffThermalExpansion = UnitUtils.ConvertFromInternalUnits(aStructuralAsset.ThermalExpansionCoefficient.X,DisplayUnitType.DUT_INV_CELSIUS);
+            material_Destination.PoissonsRatio = aStructuralAsset.PoissonRatio.X;
+            material_Destination.CompressiveYieldStrength = UnitUtils.ConvertFromInternalUnits(aStructuralAsset.ConcreteCompression, DisplayUnitType.DUT_PASCALS);
```

Comment:
Would assume this property only exist for concrete materials. Would be good to add a check what material type it is, and add the appropriate design parameters, at least for steel and concrete.

Reply:
If Materials are common for all disciplines then we need to agree which parameters will be filled in... I strongly believe the data such as Density should be copied for all materials.

Reply:
Sorry, was not clear enough! Only meant the very last row!